### PR TITLE
Add `updateMany`, `deleteMany` and `isSet` to composite types page

### LINKED ISSUE #2986: Document remaining composite types filters (Epic 3)
Document any composite types filters that didn't make it into 3.11.

- updateMany
- deleteMany
- isSet

Continuation of #2942.

### PR DESCRIPTION
**Pull request by keerlu**
## Describe this PR

WIP, currently just uncommented updateMany and deleteMany

### CODE REVIEW

File: content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx [hunk 1]
```diff
@@ -177,6 +178,18 @@ const orders = await prisma.order.findMany({
 })
 ```
 
+Use `isSet` to filter for orders where the optional `billingAddress` has been set (either to a value or to `null`):
```
Comment (dpetrick) [inline content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx:181]: "either to a value or to `null`" => "this also includes fields that are explicitly set to `null`"

File: content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx [hunk 2]
```diff
@@ -536,7 +547,7 @@ const product = prisma.product.update({
     },
   },
 })
-``` -->
+```
```
Comment (dpetrick) [inline content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx:550]: Missing deleteMany example?

**Review by dpetrick — APPROVED** [review]

File: content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx (hunk 2 above)
Comment (keerlu) [reply content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx:550]: I'm seeing an example starting line 533, 'The following example uses `deleteMany` to delete all photos with a `height` of 100:'?

File: content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx (hunk 1 above)
Comment (keerlu) [reply content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx:181]: Done

**Pull request merged**